src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Shape {
+
+    private Random rand;
+    private StringBuilder sb = new StringBuilder();
```

Comment:
sb - it's informative naming, let's improve this

Reply:
fixed!